lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -1200,6 +1201,9 @@ class MongoClientInterface {
      * @param {string} params.vFormat - object key format.
      * @param {boolean} params.needOplogUpdate - If true, the object is directly put into the collection
      * with updating the operation log.
+     * @param {boolean} params.preventConcurrentCompletion - If true, the object is only created if 
+     * completeInProgress is not already set to true. Fix this concurrent completeMultipartUpload issue : 
+     * https://scality.atlassian.net/jira/software/c/projects/OS/boards/268?selectedIssue=CLDSRV-687
```

Comment:
just ticket number of enough

```suggestion 
          * CLDSRV-687
```